recipe/conda_forge_ci_setup/feedstock_outputs.py

```diff
@@ -62,12 +62,16 @@ def request_copy(feedstock, dists, channel, git_sha=None, comment_on_error=True)
     except Exception as e:
```

Comment:
The status code of `r` isn't checked, so this has actually _relied_ on the fact that the webapp doesn't respond with JSON when there's an error: https://github.com/conda-forge/conda-forge-webservices/issues/941

Reply:
ouch. let's fix that...